frontend/src/components/forms/order/helper.ts

```diff
@@ -5,10 +5,17 @@ export function productsBySupplier (this: string, product:IProduct):boolean {
     if(!product.suppliers || !this) return false;
```

Comment:
I don't know if it's very Javascript-style to add DocStrings but it would extremely help to have some here